HIP/hip-904-frictionless-airdrop.md

```diff
@@ -0,0 +1,706 @@
+
+### Example
+
+A company called GameCo wants to airdrop tokens (e.g., unique NFTs to commemorate a specific event) to everyone who attended a game. They can send those NFTs (say, GameNFT) to every attendee without explicit signatures from the thousands of attendees. GameCo will pay all required fees to perform this airdrop. Alice, Bob, Carol, and Dave all attended this game.
+
+Alice has an account with unlimited auto-associations. When GameCo drops the GameNFT into her account, it is immediately associated and transferred. Alice now owns a GameNFT.
+
+Bob has an account with a fixed number of auto-associations, but the number of auto-associations he’s received is less than his fixed max, so he has an open slot. When GameCo drops the GameNFT into his account, it is immediately associated and transferred. Bob now owns a GameNFT.
+
+Carol has an account with auto-associations disabled. She has no open slots. When GameCo drops the GameNFT into her account, she does not immediately receive it. Her wallet alerts her to a new airdrop. She decides she wants this token. For a nominal fee, she signs a transaction to accept this airdrop. Her account is associated with GameNFT and a serial is transferred to her account. Carol now owns a GameNFT.
+
+Dave also has an account with no open slots. When GameCo drops the GameNFT into his account, he does not immediately receive it. His wallet alerts him to a new airdrop, but Dave doesn’t want it. He tells his wallet to ignore this airdrop. A couple days later, Dave regrets his decision. He checks, and the airdrop is still available, so he decides to claim it. If he wanted to, he could have ignored this airdrop until GameCo canceled the airdrop.
+
+> A wallet may or may not provide the functionality described in this HIP. It is outside the scope of the HIP to define how wallets will handle pending transfers. The described scenario is one possible workflow.
+
+
+## User stories
+
+### Crypto Transfer User Stories
+
+#### Sender
+
+1. As an hts-token-sender, I want to send the token using the CryptoTransfer transaction to a receiver who has associated with this token.
+2. As an hts-token-sender, I want to send the token using the CryptoTransfer transaction to a receiver who has not associated with this token but has an open auto-association slot.
+3. As an hts-token-sender, I understand my call will fail if I attempt to send an HTS token using the CryptoTransfer transaction to a receiver who has not associated  with this token and has no open auto-association slot.
+4. As an hts-token-sender I want to cover the custom fees (fixed, fractional, and royalty excluding fallback) of a transfer to an account that is already associated with a token to reduce the friction by a receiving account
+5. As an hts-token-sender I want to cover the fees (association & rent until the end of the first full auto renewal period) of a transfer that consumes an automatic token slot to reduce the friction by a receiving account
```

Comment:
Receiver paying was a barrier that senders little control of prior.
With this senders can cover the costs and reduce the barrier

Reply:
since no FT value is being transferred with the NFT (for example), the fixed nor fractional fee will be enforced. What custom fee would the sender cover? 

Typically in this case, where an atomic transfer is not used which includes both an NFT and some fungible token, the fallback fee would prevent the token from being transferred without the receivers signature (since they must cover the fallback fee). It looks like tokens transfers that would activate a fallback fee are disqualified from airdrop and pending state?

Reply:
David's point is fair this is already the case but the user story is restated for clarity.
The fee that a payer is now responsible for is the association fees when an automatic association is done or when an airdrop is issued.
Responsibility of other fees such as token custom has not changed.